robottelo/constants/__init__.py

```diff
@@ -230,6 +230,7 @@ class Colored(Box):
 PRODUCT_KEY_VIRT_AGENTS = 'rhva6'
 PRODUCT_KEY_CLOUD_FORMS_TOOLS = 'rhct6'
 PRODUCT_KEY_ANSIBLE_ENGINE = 'rhae2'
+PRODUCT_KEY_RHEL_EXTRAS = 'rhel7_extra'
```

Comment:
Is it possible to move this constant to the module where it is used if it is used only once?

Reply:
Not a Blocker though !